application/src/main/java/org/togetherjava/tjbot/commands/help/ImplicitAskListener.java

```diff
@@ -185,9 +185,9 @@ private static RestAction<Message> sendInitialMessage(ThreadChannel threadChanne
             .build();
 
         return MessageUtils
-            .mentionSlashCommand(originalMessage.getGuild(), HelpThreadCommand.COMMAND_NAME,
-                    HelpThreadCommand.CHANGE_SUBCOMMAND,
-                    HelpThreadCommand.Subcommand.CHANGE_CATEGORY.getCommandName())
+            .mentionSlashCommand(originalMessage.getGuild(),
+                    String.join(HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND,
```

Comment:
i dont like that everyone has to do `String.join(" ", ...)` now. and the fact that it already lead u to two bugs by forgetting `" "`.

We should probably go back and add a vararg or List overload (sorry for the back and forth) - what do u think?

Reply:
Well I just did a full search on `String.join` and I still managed to miss some, so other people will probably miss that too when adding new command mentions in the future. I'm fine going back to varargs.